### DISCUSSION TITLE
Playlist AutoPlay Not Functioning

### DISCUSSION THREAD

**Discussion by RyanGunCoyote** [mediacms-io/mediacms#867] [Q&A]
I've created a test Playlist and have the settings to loop. Once I click Play All on the playlist the video stops on the next video in the list and requires me to manually press the play button. Seeking a solution to have all the videos on a designated Playlist to continuously play from video to video on loop without manual intervention.

**Comment by mgogoulos** [comment]
Trying to reproduce on https://demo.mediacms.io/ and it works as expected for me on Chrome/Linux. What OS/browser are you using? Maybe it's something specific to that combination?  Can you try a playlist on the demo instance and see if auto play works for you? Eg try this - https://demo.mediacms.io/playlists/3hgJADjJX

**Reply by RyanGunCoyote** [reply]
I am using MacOS 13.5.1 with Safari 16.5.2 (18615.2.9.11.10). Using your demo playlist link I also experience the same issue. The next video loads but requires me to manually press the Play button. Here is my test playlist for reference.
https://tubecoyote.com/playlists/OnQldLbyH

Edit: I also tested both the demo playlist and my playlist using Firefox 116.0.2 on MacOS 13.5.1 with the same issue.

Edit 2: Also appears the same on iPhone. Perhaps a Mac related issue. It appears to work on my Windows machine with Firefox during testing.

**Reply by RyanGunCoyote** [reply]
Upon further investigation it appears to be an Auto-Play block that is enabled on Apple devices. I.E. you must go to Safari>Settings>Websites>Allow All Auto-Play for the selected domain for Apple devices using Safari or any browser that is on an Apple operating system then the Autopaly function performs as it should.

This does appear to be the default setting for YouTube on Apple devices, but for MediaCMS installations has to be manually

**Reply by mgogoulos** [reply]
Hey thanks for finding this out! Would be good to update the docs to ilustrate this behavior...
